content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/100-connection-management.mdx

```diff
@@ -85,6 +85,12 @@ If the above script runs multiple times in the context of a long-running applica
 
 ## Exit hooks
 
+<Admonition type="info">
+
+The `beforeExit` hook only applies to the [binary Query Engine](/concepts/components/prisma-engines/query-engine#configuring-the-query-engine).
```

Comment:
We should probably mention that this is since 5.0.0 (and it used to be for library engine too before.)

Reply:
Also: The binary engine is not really a thing a user will understand. Technically this feature is removed, and only for some specific case it still exists.